src/psycop_feature_generation/loaders/raw/load_visits.py

```diff
@@ -91,19 +91,20 @@ def physical_visits(
     }
 
     allowed_visit_types = ("admissions", "ambulatory_visits", "emergency_visits")
-    if visit_type not in allowed_visit_types:
+    if visit_types not in allowed_visit_types:
         raise ValueError(
             f"Invalid visit type. Allowed types of visits are {allowed_visit_types}.",
         )
 
-    if visit_type:
+    if visit_types:
         LPR3_types = {  # pylint: disable=invalid-name
```

Comment:
```suggestion
        english_to_lpr3_visit_type = {  # pylint: disable=invalid-name
```

Suggest renaming since it's a mapping :-)